scripts/train_residual_flow.py

```diff
@@ -113,79 +128,173 @@ def main(cfg):
 
     dm.setup()
 
-    network = ResidualFlow_DiffEmbTransformer(
-        emb_dims=cfg.model.emb_dims,
-        emb_nn=cfg.model.emb_nn,
-        return_flow_component=cfg.model.return_flow_component,
-        center_feature=cfg.model.center_feature,
-        pred_weight=cfg.model.pred_weight,
-        multilaterate=cfg.model.multilaterate,
-        sample=cfg.model.mlat_sample,
-        mlat_nkps=cfg.model.mlat_nkps,
-        break_symmetry=cfg.break_symmetry,
-    )
-
-    model = EquivarianceTrainingModule(
-        network,
-        lr=cfg.training.lr,
-        image_log_period=cfg.training.image_logging_period,
-        displace_loss_weight=cfg.training.displace_loss_weight,
-        consistency_loss_weight=cfg.training.consistency_loss_weight,
-        direct_correspondence_loss_weight=cfg.training.direct_correspondence_loss_weight,
-        weight_normalize=cfg.task.phase.weight_normalize,
-        sigmoid_on=cfg.training.sigmoid_on,
-        softmax_temperature=cfg.task.phase.softmax_temperature,
-        flow_supervision=cfg.training.flow_supervision,
-    )
+    if cfg.model.name in ["taxposed", "taxposed_mlat_s100", "taxposed_mlat_s256"]:
```

Comment:
might want to extract this conditional into a function above. just so that it's clear what's actually being set up in the main function. (i.e. a network, a model, a dataset, and training).

Reply:
created functions for network creation, model creation, and maybe loading weights so main function is clear